src/improvements/tasks/OPCMBaseTask.sol

```diff
@@ -73,7 +69,17 @@ abstract contract OPCMBaseTask is L2TaskBase {
             _stateInfos[parentMultisig].length == 1,
             "OPCMBaseTask: only nonce should be updated on upgrade controller multisig"
         );
```

Comment:
we can remove this use of `_stateInfos` and replace with the AccountAccessParser lib

Reply:
Addressed in: https://github.com/ethereum-optimism/superchain-ops/pull/664/commits/c908437b9e807b01b16c9867d5f9b15aa62d615b